apps.awesim.org/apps/myjobs/templates/Basic_BLAST_Job_Pitzer/blast.sh

```diff
@@ -0,0 +1,31 @@
+#PBS -l nodes=1:ppn=40
+#PBS -l walltime=1:10:00
```

Comment:
Can this run under an hour? If so it should be set to something like `00:45:00` so that debug queue would be an option.